platform-core-module/src/main/java/com/softicar/platform/core/module/page/header/PageHeaderUserPopover.java

```diff
@@ -0,0 +1,76 @@
+package com.softicar.platform.core.module.page.header;
+
+import com.softicar.platform.ajax.document.AjaxDocument;
+import com.softicar.platform.common.code.reference.point.SourceCodeReferencePoints;
+import com.softicar.platform.core.module.AGCoreModuleInstance;
+import com.softicar.platform.core.module.CoreI18n;
+import com.softicar.platform.core.module.CoreImages;
+import com.softicar.platform.core.module.CoreTestMarker;
+import com.softicar.platform.core.module.ajax.session.reset.AjaxSessionPage;
+import com.softicar.platform.core.module.page.navigation.PageNavigationController;
+import com.softicar.platform.core.module.page.service.PageServiceFactory;
+import com.softicar.platform.core.module.user.profile.UserProfilePage;
+import com.softicar.platform.core.module.web.service.WebServiceUrlBuilder;
+import com.softicar.platform.dom.elements.button.DomButton;
+import com.softicar.platform.dom.elements.popup.modal.DomPopover;
+import com.softicar.platform.dom.elements.popup.position.strategy.DomPopupBelowEventNodePositionStrategy;
+import com.softicar.platform.emf.EmfCssClasses;
+import com.softicar.platform.emf.page.IEmfPage;
+import javax.servlet.http.HttpSession;
+
+class PageHeaderUserPopover extends DomPopover {
+
+	private final PageNavigationController controller;
+
+	public PageHeaderUserPopover(PageNavigationController controller) {
+
+		this.controller = controller;
+		this.configuration.setPositionStrategy(new DomPopupBelowEventNodePositionStrategy());
+		addCssClass(EmfCssClasses.EMF_MANAGEMENT_ACTIONS_POPOVER);
```

Comment:
we should consider defining some `DomMenuPopover` or `DomActionPopover` class and CSS class

Reply:
I'd rather do that in a separate PR. We need do fix some CSS-related things regarding `DomPopover` anyway... that should not creep into this PR.

Reply:
> I'd rather do that in a separate PR.

ok, but should be part of the issue